### PR TITLE
Revisions to Networking 101 Step 4

### PR DESCRIPTION
**Pull request by klevenstein**

### CODE REVIEW

File: labs/networking-101-the-basics/4.md [hunk 1]
```diff
@@ -1,72 +1,70 @@
...
 *Various Cisco Switches*
 
-A switch:
-* Is a [Layer 2 (Data Link Layer)](https://en.wikipedia.org/wiki/Data_link_layer) device, so most interfaces do not have IPv4 addresses. Layers will be covered in another learning lab.
-* Uses its RJ45 ports to connect to other devices such as a router, switches, and other network hosts (e.g., computers and printers).
-* Uses ARP to determine with which device it should exchange data.
-* Uses microsegmentation to prevents network data collisions and improve traffic flow.
-* Has a configurable software interface for each of its RJ45 ports.
-* Supports configuration of its interfaces, for example, to create [Virtual LANs (VLANs)](https://en.wikipedia.org/wiki/Virtual_LAN) (not covered in this lesson).
+A switch is a [Layer 2 (Data Link Layer)](https://en.wikipedia.org/wiki/Data_link_layer) device, so most interfaces do not have IPv4 addresses. Note: Layers will be covered in another learning lab.
```
Comment (annegentle) [inline labs/networking-101-the-basics/4.md]: I should update the style guide to say capitalize Learning Lab as if it's a product name. I do the same for Track and Module.

File: labs/networking-101-the-basics/4.md [hunk 2]
```diff
@@ -1,72 +1,70 @@
...
+A router is a [Layer 3 (Network Layer)](https://en.wikipedia.org/wiki/Network_layer) device. Most of its interfaces have IPv4 addresses.
 
-Here is a sample configuration of a router's interfaces. Notice that each interface has an IP address.
-![](/posts/files/networking-101-the-basics/assets/images/router-config.png)
-*Sample Router Configuration*
+Routers have the following characteristics:
 
+* RJ45 ports connect to other routers or switches. Each port has a configurable software interface.
+* A routing table determines which switch or router should receive data. It routes data to the proper network device.
+* Interfaces and other features such as routing tables, and network address translation can be configured by an administrator. Configuration is not covered in this learning lab.
```
Comment (annegentle) [inline labs/networking-101-the-basics/4.md]: Same here, capitalize Learning Lab.

File: labs/networking-101-the-basics/4.md [hunk 3]
```diff
@@ -1,72 +1,70 @@
...
+* Starting from the bottom, right to left, the student network host computers connect to the RJ45 ports on the switch. Each of the portals has an interface (not shown).
+* A port on the switch connects to the router's (RTR2) RJ45 port, defined by the FastEthernet1/0 interface.
+* The router (RTR2) connects to another router (RTR1), which, in this case, is through a portal defined by a Serial0/0 interface.
+* The router (RTR2) also connects to a switch through its portal, which is defined by a FastEthernet0/0 interface.
+* The switch connects to the Instructor network host computers through its RJ45 ports. Each of these ports has an interface (not shown).
 7. Bear in mind that each of the network host devices has interfaces with defined IP addresses as well.
 
-### The Network Engineer
-Network engineers are responsible for configuring the interfaces and other features of these hardware devices. There are ways to add some of this information dynamically, but that also requires a network engineer to perform configuration tasks. Network engineers also perform all of the physical wiring, connecting devices using Ethernet cable.
+## About network engineer
```
Comment (annegentle) [inline labs/networking-101-the-basics/4.md]: engineers

**Comment by klevenstein** [comment]
changes made; I'll also update the Lab Style Guide.

**Pull request merged**